src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -340,6 +356,8 @@ public function updateAction(Request $request)
             return $this->viewHandler->handle($configuration, View::create($form, Response::HTTP_BAD_REQUEST));
         }
 
+        $this->eventDispatcher->dispatch(ResourceActions::INITIALIZE, $configuration, $resource);
```

Comment:
@pjedrzejewski there was an idea to have initialize events for all resource actions `initialize_update`, `initialize_create` etc. is it abandoned?